apps/dashboard/src/components/contract-components/hooks.ts

```diff
@@ -602,7 +591,9 @@ export function useCustomContractDeployMutation(
   const walletId = useActiveWallet()?.id;
 
   return useMutation(
-    async (data: ContractDeployMutationParams) => {
+    async (_data: ContractDeployMutationParams) => {
+      const data = { ..._data };
```

Comment:
why is the copy necessary here? do we mutate it somewhere?

Reply:
Not necessary, but it felt weird that we are mutating the form data here, so I made a copy to ensure that does not happen